### DISCUSSION TITLE
Developers' Guide link needs to be updated

### DISCUSSION THREAD

**Discussion by w4ssimh** [pyocd/pyOCD#1307] [General]
Hi,

I was hanging out in the git README.md and I noticed that the link attached to the "**Developers' guide**" is not working (pyocd.io/docs/developers-guide).

The correct link is : pyocd.io/docs/developers_guide

The same link but with _ instead of -

Kind regards,
Wassim

**Comment by flit** [comment]
Whoops, thanks!

**Reply by flit** [reply]
In fact, most of the links to the docs were broken. 😢
